codemeta.json

```diff
@@ -0,0 +1,296 @@
+            "affiliation": "Center for Computational Quantum Chemistry, University of Georgia",
+            "@id": "https://orcid.org/0000-0003-3659-0711",
+            "@id": "https://github.com/jturney"
+        },
+        {
+            "@type": "Person",
+            "givenName": "T.",
+            "additionalName": "Daniel",
+            "familyName": "Crawford",
+            "affiliation": "Department of Chemistry, Virginia Institute of Technology",
+            "@id": "https://orcid.org/0000-0002-7961-7016",
+            "@id": "https://github.com/lothian"
+        },
+        {
+            "@type": "Person",
+            "givenName": "C.",
+            "additionalName": "David",
+            "familyName": "Sherrill",
+            "affiliation": "Center for Computational Molecular Science and Technology, Georgia Institute of Technology",
+            "@id": "https://orcid.org/0000-0002-5570-7666",
+            "@id": "https://github.com/CDSherrill"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Peter",
+            "familyName": "Kraus",
+            "affiliation": "Liebniz University, Hannover, Germany",
+            "@id": "https://orcid.org/0000-",
+            "@id": "https://github.com/PeterKraus"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Holger",
+            "familyName": "Kruse",
+            "affiliation": "Institute of Biophysics, Czech Republic",
+            "@id": "https://orcid.org/0000-",
+            "@id": "https://github.com/hokru"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Marvin",
+            "additionalName": "H.",
+            "familyName": "Lechner",
+            "affiliation": "Department of Chemistry, Technical University of Munich",
+            "@id": "https://orcid.org/0000-0002-1676-2718",
+            "@id": "https://github.com/mhlechner"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Matthew",
+            "additionalName": "C.",
+            "familyName": "Schieber",
+            "affiliation": "Center for Computational Molecular Science and Technology, Georgia Institute of Technology",
+            "@id": "https://orcid.org/0000-",
+            "@id": "https://github.com/schiebermc"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Robert",
+            "additionalName": "A.",
+            "familyName": "Shaw",
+            "affiliation": "University of Sheffield",
+            "@id": "https://orcid.org/0000-",
+            "@id": "https://github.com/robashaw"
+        }
+    ],
+    "developmentStatus": "active",
+    "downloadUrl": "https://github.com/caltechlibrary/datatools/archive/v0.0.24.zip",
+    "keywords": [
+        "csv",
+        "json",
+        "xlsx",
+        "golang",
+        "bash"
+    ],
+    "maintainer": "https://orcid.org/0000-0003-0900-6903",
+    "programmingLanguage": [
```

Comment:
Add `C` and maybe Fortran?

Reply:
will do. and ping anyone nearby who doesn't have their orcid filled in